src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js

```diff
@@ -8,12 +8,13 @@ import Image from "../../../../components/image";
 import HowIntegrationWorks from "./howItWork";
 import { IntegrationPageWrapper } from "./individual-integrations.style";
 
+
 const IndividualIntegrations = ({ data }) => {
   const { frontmatter, body } = data.mdx;
   return (
     <IntegrationPageWrapper>
       <section className="herosection">
-        <Image {...frontmatter.integrationIcon} imgStyle={{ objectFit: "contain" }} alt={frontmatter.title}/>
+        <Image {...frontmatter.integrationIcon} imgStyle={{ height: "90px" }} alt={frontmatter.title}/>
```

Comment:
I see you are setting the height in CSS as well, do we need it here too?
And,

Reply:
No, we don't need it here. 😅

Reply:
Then, please do the needful :)

Reply:
Sure.